### DISCUSSION TITLE
Saving & Reloading transpiled output

### DISCUSSION THREAD

**Discussion by darrinm** [codesandbox/sandpack#259] [Q&A] [answered]
I'd like to save the transpiled output of Sandpack (seems pretty straightforward) and subsequently load it in an iframe. A sort of "player" for pre-built Sandboxes that wouldn't have all the building/bundling overhead. I get that I could use conventional webpack, etc to produce a static build but I'd like to do it all in the browser.

Any ideas where to start?

**Comment by danilowoz** [comment, marked as answer]
Hey @darrinm, thanks for raising this issue. But first of all, I would like to get to understand what is the problem exactly you're trying to solve. 

If you would like to skip the transpile step in order to improve the loading/refreshing sandbox time, we have already addressed an internal solution for it, which aims to cache all compiled files and share them between sandboxes with the same dependencies. Although it looks very promising, we still don't know when we can ship it for all sandboxes.

Let me know if I understood your question correctly.

**Reply by danilowoz** [reply]
Moved the discussion to https://github.com/codesandbox/sandpack/issues/295
